### ISSUE TITLE
Not finding statuses

### ISSUE THREAD

**Issue by RebeccaStevens** [WyriHaximus/github-action-wait-for-status#104]
`[2022-04-01 23:06:49.946339] wait.WARNING: No statuses found, assuming success`

https://github.com/jonaskello/eslint-plugin-functional/runs/5795726153?check_suite_focus=true

I can't figure out why no statuses are being found. Could you help me?

Also can I change `assuming success` to `marking as failure`? If no statuses were found, it's probably an error rather than just that there are none.

**Comment by WyriHaximus** [comment]
Yeah will have a look, are you using reusing workflows like in this issue? https://github.com/WyriHaximus/github-action-wait-for-status/issues/103

**Comment by RebeccaStevens** [comment]
Yeah, it looks very similar.

**Comment by WyriHaximus** [comment]
@RebeccaStevens [`v1.5.0`](https://github.com/WyriHaximus/github-action-wait-for-status/releases/tag/v1.5.0) is out with a potential fix for this in it. Could you let me know if this fixes it for you? Also, be aware that GitHub has two systems with regard to PR's in the API. Checks and Statuses, it's been moving away from statuses toward checks but this action still has to check statuses in case something sets a status on a commit/PR. If you have a to wait a bit for a status to show up @DarkKirb PRed a feature to do this: https://github.com/WyriHaximus/github-action-wait-for-status/pull/107
